draft-ietf-scitt-software-use-cases.md

```diff
@@ -203,14 +203,14 @@ There is no standardized way to:
 
 ## Promotion of a Software Component by Multiple Entities
 
-A software component source (e.g., a library) released by a certain original producer is becoming popular.
-The released software component source is accompanied by a statement of authenticity (e.g., a detached signature).
+A software component (e.g., a library) released by a certain original producer is becoming popular.
```

Comment:
Minor, non-blocking: given the usage of the word source and source code in software supply chain, I found this wording a little confusing in my nit-picky ways. And to be fair: a component is often consumed compiled/transpiled/finalized and not in "source" form, so I'd recommend not muddying the waters here. (I know that is not 100% true in all cases, but the murkiest cases are the ones I refer to where source is not 1:1 equivalent to the resulting artifact, or why do we need transparency and integrity then, to be rhetorical about it?